appveyor.yml

```diff
@@ -0,0 +1,68 @@
+build: false
+
+init:
+  - SET PATH=C:\Program Files\OpenSSL;c:\tools\php;c:\cygwin\bin;%PATH%
+
+  # uncomment to allow remote desktop connection
+  #- ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+on_finish:
+  # uncomment to show remote desktop connection details and block build
+  #- ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+
+services:
+  - mysql
+
+environment:
+  SYMFONY_DEPRECATIONS_HELPER: weak
+  JACKRABBIT_VERSION: '2.12.0'
+  SYMFONY__DATABASE__PASSWORD: Password12!
+
+platform: x86
+
+cache:
+  # cache composer downloads
+  - '%LOCALAPPDATA%\Composer\files'
+
+  # jackrabbit will be re-downloaded when jackrabbit.sh is modified.
+  - jackrabbit-standalone-%JACKRABBIT_VERSION%.jar -> bin\jackrabbit.sh
+
+
+install:
+  # install SSL and php
+  - cinst -y OpenSSL.Light
+  - cinst -y php
+
+  # configure PHP and enable extensions.
+  - cd c:\tools\php
+  - copy php.ini-production php.ini /Y
+  - echo date.timezone="UTC" >> php.ini
+  - echo extension_dir=ext >> php.ini
+  - echo extension=php_gd2.dll >> php.ini
+  - echo extension=php_openssl.dll >> php.ini
+  - echo extension=php_mbstring.dll >> php.ini
+  - echo extension=php_intl.dll >> php.ini
+  - echo extension=php_pdo_sqlite.dll >> php.ini
+  - echo extension=php_pdo_mysql.dll >> php.ini
+  - echo extension=php_curl.dll >> php.ini
+  - echo extension=php_fileinfo.dll >> php.ini
+  - echo memory_limit=1G >> php.ini
+  - appveyor DownloadFile https://phar.phpunit.de/phpunit.phar
+  - ps: echo ("@php c:\tools\php\phpunit.phar %*") | Out-File phpunit.bat -encoding ascii
+
+  # install and start jackrabbit
+  - cd %APPVEYOR_BUILD_FOLDER%
+  - ls 
+  - if not exist jackrabbit-standalone-%JACKRABBIT_VERSION%.jar appveyor DownloadFile http://archive.apache.org/dist/jackrabbit/%JACKRABBIT_VERSION%/jackrabbit-standalone-%JACKRABBIT_VERSION%.jar
+  - ls
+  - ps: Start-Process ("jackrabbit-standalone-" + $env:JACKRABBIT_VERSION + ".jar")
+  # sleep to allow jackrabbit to start - maybe not necessary as we have some time while composer installs the deps.
+  - ps: Start-Sleep -s 1
```

Comment:
Basically we are saying that as long as composer doesn't install very
quickly and that there are enough tests before Jackrabbit is invoked for
the first time then Jackrabbit *should* be ready. Its hardly a
guarantee - its just incidental that the test process will allow time
for Jackrabbit to start.

Saying that, I will just remove it.

On Thu, Mar 31, 2016 at 04:11:13AM -0700, Daniel Rotter wrote:
>    In [1]appveyor.yml:
> 
>  > +  - echo extension=php_pdo_sqlite.dll >> php.ini
>  > +  - echo extension=php_pdo_mysql.dll >> php.ini
>  > +  - echo extension=php_curl.dll >> php.ini
>  > +  - echo extension=php_fileinfo.dll >> php.ini
>  > +  - echo memory_limit=1G >> php.ini
>  > +  - appveyor DownloadFile https://phar.phpunit.de/phpunit.phar
>  > +  - ps: echo ("@php c:\tools\php\phpunit.phar %*") | Out-File phpunit.bat -encoding ascii
>  > +
>  > +  # install and start jackrabbit
>  > +  - cd %APPVEYOR_BUILD_FOLDER%
>  > +  - ls
>  > +  - if not exist jackrabbit-standalone-%JACKRABBIT_VERSION%.jar appveyor DownloadFile http://archive.apache.org/dist/jackrabbit/%JACKRABBIT_VERSION%/jackrabbit-standalone-%JACKRABBIT_VERSION%.jar
>  > +  - ls
>  > +  - ps: Start-Process ("jackrabbit-standalone-" + $env:JACKRABBIT_VERSION + ".jar")
>  > +  # sleep to allow jackrabbit to start - maybe not necessary as we have some time while composer installs the deps.
>  > +  - ps: Start-Sleep -s 1
> 
>    So it is really necessary? And not possible to remove at all?
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu/pull/2191#discussion_r58036747
>    2. https://github.com/sulu/sulu/pull/2191/files/a16379d74531ab180327be25d66db6709f28cccc#r58036747
>    3. https://github.com/sulu/sulu/pull/2191/files/a16379d74531ab180327be25d66db6709f28cccc#r58036747